src/v2/Apps/Conversation/Components/ConversationList.tsx

```diff
@@ -45,7 +45,7 @@ const ConversationList: React.FC<ConversationsProps> = props => {
     .indexOf(selectedConversationID)
 
   const loadMore = () => {
```

Comment:
there is some repetition in force with this pattern, specially in conversations, latter we could share this code with other components transforming it in a hook or more than one